FAQ - What if I accidentally accept unwanted changes. Can I delete th… (#444)
chromaui/chromatic-docs: src/content/snapshot/branching-and-baselines.md

</details>

<details>
  <summary>What if I accidentally accept unwanted changes. Can I delete a build?</summary>

> Chromatic does not currently support deleting individual builds within an app. Builds are integral to Chromatic's baseline algorithms, making deleting individual links in the chain complex and impossible at the moment.

Reviewer: ```suggestion
Chromatic does not currently support deleting individual builds within a proejct. Builds are integral to Chromatic's baseline algorithms, making deleting individual links in the chain complex and impossible at the moment.
```
